doc/source/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Changelog
 
     - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
 
+    - Add support for splitting epochs into multiple files while saving by `Mainak Jas`_ and `Alexandre Gramfort`_
```

Comment:
Might as well be fancy and do `:func:Epochs.save` or whatever will make a link to the right place in the docs. I'm not sure how to reference methods actually.